src/components/search-bar/search.component.jsx

```diff
@@ -0,0 +1,24 @@
+import searchlogo from '../../assets/search.svg';
+import "./search.style.css"
+
+const SearchBar = ({searchinput}) => {
+  
+
+      const eventkey = (event) => {
+      if(event.keyCode === 13){
+        document.getElementById("Text").classList.add("enteredtext");
+        setTimeout(function(){document.getElementById("Text").classList.remove("enteredtext")
+      },500)
+    }      
+    }
```

Comment:
- the function name is very general and doesn't give any hint about what does it do for us.
- manipulating the DOM this way is not recommended in React you may use `useRef` or create a piece of state to handle this and add the class name conditionally.